src/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -38,9 +38,9 @@ public override IFakeOptionsForWrappers<T> Wrapping(T wrappedInstance)
             return this.fakeOptions.Wrapping(wrappedInstance);
         }
 
-        public override IFakeOptions<T> WithAdditionalAttributes(IEnumerable<CustomAttributeBuilder> customAttributeBuilders)
+        public override IFakeOptions<T> WithAdditionalAttributes(params Expression<Func<Attribute>>[] customAttributeExpressions)

```

Comment:
I'm not particularly happy about the name either... `attributes` would work, I guess, but it's a bit misleading, since we're not actually passing attributes.